app/assets/stylesheets/components/_button.scss

```diff
@@ -66,7 +66,14 @@
   background-color: white;
   filter: drop-shadow(0px 4px 4px rgba(0, 0, 0, 0.15));
   font-weight: bold;
-  text-decoration: none
+  text-decoration: none;
+  cursor: pointer;
```

Comment:
Can achieve this via HTML in a better way by changing the div with `onclick="window.location.href = '/trades/new'"` to an anchor tag with the rails `link_to` helper method.